party3.tex

```diff
@@ -0,0 +1,33 @@
+
+
+Though the dev guide has a procedure \footnote{\url{https://developer.lsst.io/stack/packaging-third-party-eups-dependencies.html}}
+for making a third part package for inclusion in the pipelines distrib we have no official policy for how we accept and deal with such packages.
+
+
+
+There are several reasons to include third part code in our distributions:
+\begin{enumerate}
+\item There are packages which we use in our code e.g. starlink\_ast.  These are standard dependencies.  Currently these are put in lsst github org and probably should not be. Apart from that this is a well understood reason for third party code. \label{item:depend}
+\item There are packages which we want to use when working with the deployed software. AstroPy might be such an example - we don't need it to build or run our code but its sure handy for interaction with the data.\label{item:want}
+\item A collaborator has a package which depends on our code and is intended for some form of post processing. Another example here would be an algorithm we want to use such as ngmix which we do not necessarily need to support. This is what most people on LSST are thinking about under the topic of third party software. \label{item:colab}
+\end{enumerate}
+
+
+In all three cases the DMCCB should decide if a package is to be included in the distribution. We currently only distribute pipelines and this is the code base most likely to be affected here. One could see a need to distribute AP separately from DRP in production - the principle would remain the same a package may be included in one or more distributions. The DMCCB needs to control the list of such packages.
+
+We currently handle dependencies such as in type \ref{item:depend} above using using EUPS and soon via conda. We could use the same mechanism for the other types of packages also,
+we should require the third party contributed packages are conda installable.
+
+\subsection{Support}\label{sec:support}
+We should take care to clarify that inclusion of a package in an LSST distribution does not imply support for that package.
+Packages included especially of the type \ref{item:colab} above, could be included as is not even necessarily built by our CI system nor do they need to conform to any LSST rules apart from licensing.
+These should not live in the LSST github org. We currently conflate inclusion in the distrib with inclusion in the org.
+We should create another github org such as lsst-community for these packages. Though if we use something like conda to install the requisite packages this is not strictly necessary.
+\footnote{Even if we do support packages like starlink\_ast we should not put them under the LSST org I feel}.
```

Comment:
All of the above about where packages live seems to me to hinge on what you mean by “included in our distributions”.

Reply:
added "By support we mean that LSST personnel will not necessarily fix this package if has a problem.
"

Reply:
ok